tests/integration/defs/perf/disagg/utils/artifactory_uploader.py

```
@@ -0,0 +1,297 @@
+                    print(f"   Success (HTTP {status_code})")
+                    return True
+                else:
+                    print(f"   Failed (HTTP {status_code})")
+                    return False
+        
+        except urllib.error.HTTPError as e:
+            print(f"   HTTP Error: {e.code} - {e.reason}")
+            try:
+                error_body = e.read().decode('utf-8')
+                print(f"   Response: {error_body}")
+            except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)